site_root/input/resources/profiles/StructureDefinition-kfdrc-condition.json

```diff
@@ -0,0 +1,54 @@
+{
+    "resourceType": "StructureDefinition",
+    "id": "kfdrc-condition",
+    "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition",
+    "version": "0.1.0",
+    "name": "kfdrc-condition",
+    "title": "KFDRC Condition",
```

Comment:
Since Meen used `Kids First DRC <thing>` can you please change this one so its consistent?